### PR TITLE
Refactor debug logs in transactions module

### LINKED ISSUE #26367: Remove redundant test logs in ballerina transaction manager.
**Description:**
$subject for the following logs.

Checked with @afkham who was the author of the code and those are INFO logs added to validate the behaviour. We can safely remvoe them as it is not meaningful for the end user. 

```
2020-03-18 12:36:26,983 INFO  [ballerina/transactions] - Created transaction: 17130bbc-28e0-414d-b0a0-e20efccdacec
Inserted count: 1
2020-03-18 12:36:26,999 INFO  [ballerina/transactions] - Running 2-phase commit for transaction: 17130bbc-28e0-414d-b0a0-e20efccdacec:$anon$.$0
```

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by gimantha**
Fixes #26367

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal [hunk 1]
```diff
@@ -15,6 +15,7 @@
 // under the License.
 
 import ballerina/log;
+import ballerina/io;
```
Comment (MaryamZi) [inline stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal:18]: Shall we order the imports alphabetically?

File: stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal [hunk 2]
```diff
@@ -96,27 +97,27 @@ type RemoteParticipant object {
         // to prepare a participant
         boolean successful = true;
 
-        log:printDebug("Preparing remote participant: " + self.participantId);
+        log:printDebug(io:sprintf("Preparing remote participant: %s", self.participantId));
```
Comment (MaryamZi) [inline stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal:100]: Even with this approach, the argument is evaluated, right? We need to pass a function pointer. 

e.g.,
```suggestion
        log:printDebug(() => io:sprintf("Preparing remote participant: %s", self.participantId));
```

or
```suggestion
        log:printDebug(function () {
            io:sprintf("Preparing remote participant: %s", self.participantId)
        });
```

File: stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal (hunk 1 above)
Comment (gimantha) [reply stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal:17]: fixed

File: stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal (hunk 2 above)
Comment (gimantha) [reply stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal]: fixed

**Review by mohanvive — APPROVED** [review]

**Pull request merged**